docker/compose.yml

```diff
@@ -748,8 +748,8 @@ services:
       - ./config/zkbesu-shomei/zkbesu-config.toml:/var/lib/besu/zkbesu-config.toml:ro
       - ./config/zkbesu-shomei/log4j-staterecovery.xml:/var/lib/besu/log4j.xml:ro
       - ./config/linea-local-dev-genesis-PoA-besu.json/:/var/lib/besu/genesis.json:ro
-      - ../state-recovery/besu-plugin/build/libs/linea-staterecovery-plugin-0.0.1-rc2.jar:/opt/besu/lib/linea-staterecovery-plugin-0.0.1-rc2.jar
-      - ../state-recovery/besu-plugin/build/libs/linea-staterecovery-plugin-0.0.1-rc2.jar:/opt/besu/plugins/linea-staterecovery-plugin-0.0.1-rc2.jar
+      - ../state-recovery/besu-plugin/build/libs/linea-staterecovery-besu-plugin-SNAPSHOT.jar:/opt/besu/lib/linea-staterecovery-besu-plugin-SNAPSHOT.jar
```

Comment:
Does it mean the staterecovery plugin need to be built beforehand in order to run this docker compose locally?

Reply:
Not necessarily, The compose can run just fine but won't have the state recovery plugin.